src/ui/commands/ServerCommands.ts

```
@@ -0,0 +1,690 @@
+import * as vscode from 'vscode';
+import { uiStateManager } from '../core/UIStateManager';
+import { themeService } from '../core/ThemeService';
+
+/**
+ * Commands for Server control and utilities
+ */
+export class ServerCommands {
+  private disposables: vscode.Disposable[] = [];
+  private serverInstance?: { close: () => void };
+
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)